### PR TITLE
Motors tab: moved motor tool buttons away from the bottom panel

### PR DESCRIPTION
**Pull request by limonspb**
Moved motor reorder and motor direction buttons.

### Bug:

Noticed on some phones there is no enough room for three buttons at the bottom panel. It looks like that (thanks to Andy for screenshot):
image

### The solution is to move these buttons to the first panel:
image

### CODE REVIEW

**Review by McGiverGim — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by chmelevskij — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**
